### DISCUSSION TITLE
Custom page or forms in Admin site

### DISCUSSION THREAD

**Discussion by MostafaBelihi** [OrchardCMS/OrchardCore#15859] [Q&A]
Is there a way to create a page or a form in the Admin site?

I need to add some of forms that serve a company's employee so that I can control it with login and roles. This is better to be isolated from the public site that is represented by the CMS site in OrchardCore.

I can create admin menus that points to nodes within Admin pages, but I miss this part.

Is there a way around?

**Comment by ns8482e** [comment]
See Samples in Demo module

**Comment by MostafaBelihi** [comment]
I think that this tutorial is playing around it, am I right?

https://docs.orchardcore.net/en/dev/docs/guides/add-admin-menu/

**Comment by hishamco** [comment]
> I think that this tutorial is playing around it, am I right?

This tutorial for adding admin menu

**Comment by MostafaBelihi** [comment]
But it added cshtml attached to them.

**Comment by hishamco** [comment]
Ya, so whenever you click the menu item it will open the associated view, basically what prevent you to add views directly into `Views` folder?

**Comment by MostafaBelihi** [comment]
Nothing prevents me, but I need to link the view in Admin menus, and the mentioned tutorial seems helping in that.

Is there any guide to accompany the samples done in the Demo module?

**Comment by hishamco** [comment]
> but I need to link the view in Admin menus

So you can use admin menus as its mentioned in the link above
